### ISSUE TITLE
QT/SSL errors on Debian

### ISSUE THREAD

**Issue by mrkafk** [Pext/Pext#129]
**Describe the bug**
Cannot download and install any module on Debian 9, GNOME.

**To Reproduce**
1. Create python 3.7 virtualenv
2. Start pext
3. Try installing App Launcher module
4. See error:

```
(pext37) mark@titan ~/venvs % pext
python3-opengl is not installed. If Pext fails to render, please try installing it. See https://github.com/Pext/Pext/issues/11.
Using locale: en_US (system locale)
Localization loaded: True
file:///home/mark/venvs/pext37/lib/python3.7/site-packages/pext/qml/main.qml:744:13: QML Button: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
file:///home/mark/venvs/pext37/lib/python3.7/site-packages/pext/qml/main.qml:755:13: QML TextField: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
file:///home/mark/venvs/pext37/lib/python3.7/site-packages/pext/qml/main.qml:862:13: QML Text: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qt.network.ssl: QSslSocket: cannot resolve SSLv3_client_method
qt.network.ssl: QSslSocket: cannot resolve SSLv3_server_method

Usage:   
  /home/mark/venvs/pext37/bin/python3.7 -m pip install [options] <requirement specifier> [package-index-options] ...
  /home/mark/venvs/pext37/bin/python3.7 -m pip install [options] -r <requirements file> [package-index-options] ...
  /home/mark/venvs/pext37/bin/python3.7 -m pip install [options] [-e] <vcs project url> ...
  /home/mark/venvs/pext37/bin/python3.7 -m pip install [options] [-e] <local project path> ...
  /home/mark/venvs/pext37/bin/python3.7 -m pip install [options] <archive url/path> ...

no such option: --system

```

**Expected behavior**
The module is installed

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Debian 9, Gnome

% lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.5 (stretch)
Release:	9.5
Codename:	stretch

 - Pext Version 0.19

**Additional context**
Before it was even worse: No SSL worked in QT. I tried to fix that with:

apt install libssl1.0-dev 

But there's still some ssl error:

	qt.network.ssl: QSslSocket: cannot resolve SSLv3_client_method
	qt.network.ssl: QSslSocket: cannot resolve SSLv3_server_method

Additionally, as you can see, pip is called with --system flag that it does not understand.

% pip -V
pip 18.0 from /home/mark/venvs/pext37/lib/python3.7/site-packages/pip (python 3.7)

**Comment by TheLastProject** [comment]
Thank you for opening this issue!

The fact that you got to a pip command means that it was able to download the module. If the install was requested through the "From online module list" dialog and not the "From URL" one, that's not something we'd have to worry about probably, it probably just used another cipher then.

The pip issue is interesting. I forgot that a venv uses another pip than the system pip, which would make the cheap Debian workaround hack fail: https://github.com/Pext/Pext/blob/08a346f98a177dd21c063b5ff6f811c55adac01a/pext/__main__.py#L980. Do you perhaps have a suggestion on a better check to put there? Because Debian's system pip refuses to work without adding the --system flag, sadly.

**Comment by TheLastProject** [comment]
This should be fixed in 11a6dea3dab8e2bfe934ba65a60a2bd6590c5f8f. I don't have a Debian machine around so if you get the chance to test this I'd love to hear from you if it actually does :)

Closing this issue now under the assumption it probably is fixed, please reopen if I'm wrong.
